CONTRIBUTING.md

```diff
@@ -0,0 +1,55 @@
+# Contributing
+
+You can participate in Genji in several ways, there is so much to do to make this project awesome!
+
+  - [Reporting a bug](#reporting-a-bug)
+  - [Proposing an SQL feature or a general design change](#proposing-an-sql-feature-or-a-general-design-change)
+  - [Other contributions](#other-contributions)
+
+The goal of this document is to provide guidelines to contributors to make working with Genji a smooth experience.
+Genji is a complex project that has many moving parts and we want to make sure that everyone's working towards the same goal.
+
+We want to avoid situations where you would put energy on a contribution that gets rejected because it doesn't match the direction
+this project is taking.
+
+## Reporting a bug
+
+We use Github issues to track bugs. Report a bug by [opening an issue](https://github.com/genjidb/genji/issues/new), it's that easy!
+
+Make sure you give enough context for us to reproduce the bug:
+
+- The version of Genji you are using
+- The observed behavior
+- The expected behavior
+- Ideally, some code like a main file or a Go [playground](https://play.golang.org/) link
```

Comment:
We can also create [issue templates](https://github.blog/2016-02-17-issue-and-pull-request-templates) (bug report, feature request, etc). See https://docs.github.com/en/free-pro-team@latest/github/building-a-strong-community/about-issue-and-pull-request-templates

Reply:
Very usefull !

Reply:
Yes definitely, I opened an issue 👍🏼